ddp-workspace/projects/ddp-mbc/src/app/app.module.ts

```diff
@@ -57,9 +57,9 @@ tkCfg.mailingListDialogUrl = 'updates';
 tkCfg.phone = '617-800-1622';
 tkCfg.infoEmail = 'info@mbcproject.org';
 tkCfg.dataEmail = 'data@mbcproject.org';
-tkCfg.twitterAccountId = 'mbc_project';
-tkCfg.facebookGroupId = 'The-Metastatic-Breast-Cancer-Project-852059048224675';
-tkCfg.instagramId = 'mbc_project';
+tkCfg.twitterAccountId = 'count_me_in';
+tkCfg.facebookGroupId = 'https://www.facebook.com/joincountmein';
```

Comment:
Actually, it should be just `joincountmein`.
Nevertheless, it works because it uses a default value from i18n/en.json  
` "FacebookLink": "https://www.facebook.com/joincountmein"`

Reply:
Just wondering should we change a default value for Spanish version (i18n/es.json) because it has another URL
` "FacebookLink": "https://www.facebook.com/mbcprojectenespanol"`